CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.6.0 (2020-10-15)
```

Comment:
Looks like we missed this review yesterday and technically the date should be 2020-10-16. Not sure how much that really matters, so I'm noting it in case you want to fix, but I'll also go ahead and approve in case you want to ignore and merge as is.